CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 /backend/remote-state/azure   @terraform-azure
 /backend/remote-state/gcs     @terraform-google
 /backend/remote-state/s3      @terraform-aws
+/backend/remote-state/cos     @terraform-tencentcloud
```

Comment:
This GitHub group does not exist/this should be a handle that someone could mention in GitHub, for example, `@hashicorp/terraform-core`. I will update this now actually, as the handles listed here are not correct for the backends listed here already (they're missing the `@hashicorp/` namespace).

Reply:
Since there is no groups named `terraform-tencentcloud`, Is it ok that I write it as `@likexian`?

Reply:
Yes! That would work